.gitignore

```diff
@@ -9,4 +9,3 @@ scripts/expected.go
 scripts/output.go
 magic/magic.go
 crit/images/*.pb.go
-crit/images/*.proto
```

Comment:
No need to squash the code into a single commit. The reason for applying the changes in the previous commit is to make the pull request easier to review. Thus, each logical change is in a [separate commit](https://github.com/checkpoint-restore/criu/blob/criu-dev/CONTRIBUTING.md#separate-your-changes). For example, it would good to add the *.proto files in a separate commit.